src/models/faraday/model.py

```diff
@@ -132,6 +138,30 @@ def __init__(
         self.lower_quantile = lower_quantile
         self.upper_quantile = upper_quantile
 
+        if differential_privacy:
+
+            if max_grad_norm is None:
```

Comment:
in terms of checking params, can `epsilon` be any value?  I thought it needed to be positive for some reason...

Reply:
Yeah it has to be, but with invalid value Opacus will throw up an error on its own that I don't think we need to check. Opacus allows valid positive float values of `max_grad_norm` but we should stick it to max 1.0 as good privacy practice.